crates/layout/tests/layout_tests.rs

```diff
@@ -129,10 +129,14 @@ fn add_view_to_layout(
 fn load_doc() -> std::io::Result<SerializedDesignDoc> {
```

Comment:
This function looks like something that should be generic so that it can be re-used in other tests. But that's out of scope for this PR.

Reply:
Agreed.